packages/app-mobile/components/screens/ConfigScreen.tsx

```diff
@@ -500,7 +518,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 				</View>
 			);
 		} else if (md.type === Setting.TYPE_STRING) {
-			if (md.key === 'sync.2.path' && Platform.OS === 'android' && Platform.Version > 28) {
+			if (md.key === 'sync.2.path' && shim.fsDriver().isUsingAndroidSAF()) {
```

Comment:
Is this "isUsingAndroidSAF" backward compatible? What does "SAF" mean anyway? I'm not a fan of cryptic abbreviations

Reply:
SAF stands for Storage Access Framework, a mechanism to access files on Android. It was introduced in Android 4.4. On newer Android versions this is the only way an application without special permission can access files.

Reply:
yeah It's sotrage access framework
if It's not good we can change the name, but I dont have any better idea
and I did not understand what you mean by backward compatible. I just moved the exact same check inside `fsDriver().isUsingAndroidSAF()`.